get_kserve_manifests.sh

```
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Collect manifests for kserve-module-operator (local development).
+# Reads build/manifests-config.yaml and gathers manifests into opt/manifests/.
+#
+# For production builds, Konflux prefetch-manifests task handles this
+# via the same manifests-config.yaml.
+
+SCRIPT_DIR="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)"
+source "${SCRIPT_DIR}/hack/setup/common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./hack/setup/common.sh was not specified as input (see shellcheck -x).

(SC1091)